src/main.ts

```diff
@@ -73,5 +73,6 @@ export async function createProject(options: Options) {
   } catch (error) {
     console.log("%s Error occurred", chalk.red.bold("ERROR"), error);
     console.log("%s Exiting...", chalk.red.bold("Uh oh! 😕 Sorry about that!"));
+    throw error;
```

Comment:
Ohh since here we are throwing the error forward just throw it forward without logging it, and in `cli.ts` we move this lines? 

```ts
 console.log("%s Error occurred", chalk.red.bold("ERROR"), error);
console.log("%s Exiting...", chalk.red.bold("Uh oh! 😕 Sorry about that!"));
``` 

Because I feel `Uh oh! 😕 Sorry about that! Exiting...` should come very last 

Currently if error occurs in one of `tasks` of `main.ts` we get: 

<img width="1264" height="523" alt="Image" src="https://github.com/user-attachments/assets/69cec05b-f1f8-41fa-a916-3b2bbd4283d6" />

Reply:
Thanks! Moved logs from `main.ts` to `cli.ts` and removed try catch in `main.ts` since we don't need it anymore

<img width="1090" height="419" alt="image" src="https://github.com/user-attachments/assets/5a205906-42c9-4b33-ad0e-dc4c83e57ff1" />